### PR TITLE
Update zenodo for 5.1

### PR DESCRIPTION
**Pull request by dstansby**
👋 hello new contributors! I am updating the list of authors for the zenodo entry (see https://zenodo.org/record/3779284 for a previous example). I have mentioned you because you are a new contributor since sunpy 5.0.

If you would like, please leave a comment below with:

The preferred name you would like to be listed as
Your ORCID
Your affiliation

I will then manually add this to the information. Note that none of this is compulsory!

@TrestanSimon @hannahc243 @braingram @kurtmckee @PaulJWright @svank @STBadman

### CODE REVIEW

**Comment by STBadman** [comment]
Thank you @dstansby ! could you list me as:

Samuel T. Badman
ORCID: 0000-0002-6145-436X
Affil: Center for Astrophysics | Harvard & Smithsonian

**Comment by braingram** [comment]
Thanks!

Brett J Graham
ORCID: 0000-0001-6315-4507
Affil: Space Telescope Science Institute

**Comment by svank** [comment]
Thanks!

Samuel J. Van Kooten
0000-0002-4472-8517
Southwest Research Institute

**Comment by TrestanSimon** [comment]
Thank you!

Trestan F. Simon
0009-0000-3029-8619
No affiliation

**Comment by PaulJWright** [comment]
Thanks @dstansby.

Paul J. Wright
0000-0001-9021-611X
Dublin Institute for Advanced Studies

**Comment by kurtmckee** [comment]
Thanks @dstansby!

Kurt McKee
0000-0002-8547-8489
University of Chicago

**Comment by hannahc243** [comment]
Thanks David. Could you include me with the following details:

Hannah Collier
0000-0001-5592-8023
Fachhochschule Nordwestschweiz and ETH Zürich
________________________________

File: .zenodo.json [hunk 1]
```diff
@@ -740,7 +747,9 @@
             "name": "OussCHE"
         },
         {
-            "name": "Paul Wright"
+            "name": "Paul J. Wright",
+            "orcid": "0000-0001-9021-611X",
+            "affiliation": " Dublin Institute for Advanced Studies"
```
Comment (kurtmckee) [inline .zenodo.json:752]: Nit: extra whitespace (`" Dublin"`)

**Review by nabobalis — APPROVED** [review]

**Pull request merged**